src/core/ui.js

```diff
@@ -69,7 +69,10 @@ respecPill.addEventListener("click", e => {
   }
   respecPill.setAttribute("aria-expanded", String(menu.hidden));
   menu.hidden = !menu.hidden;
+  trapFocus(menu);
```

Comment:
Adding event listener every time when clicking pill doesn't sound good... 👀

Reply:
Agree... it needs unregister itself later when closing happens.

Reply:
Can we modify this to be called just after line 49? That won't require un-registration then.